lib/nelmon/cli/check_admin_up_oper_down.py

```diff
@@ -24,6 +25,17 @@ def main():
                                   help='Return Warning if interfaces are down')
     argparser.parser.add_argument('-c', action='store_true',
                                   help='Return Critical if interfaces are down')
+    argparser.parser.add_argument('-d', '--descr', dest='ifdescr_arg',  default=None, const=None,
+                                  help='Search over Interface descr with regex'
+                                       'example: GigabitEthernet(\d+)/0/(4[78]|[5][0-2])'
+                                       'matches any of: GigabitEthernetx/0/47,48,50,51,52')
+    argparser.parser.add_argument('-al', '--alias', dest='ifalias_arg',  default=None, const=None,
+                                  help='Search over Interface alias with regex'
+                                       'example: UPLINK'
+                                       'matches any interfaces with keyword UPLINK on its alias')
+    argparser.parser.add_argument('-id', '--ignore_descr', dest='ifdescr_ignore_arg',  default=None, const=None,
```

Comment:
Although I know all the "good letters" are already taken I'd like the arguments to include only one letter. Can you think of any other alternatives instead of -al and -id?

Reply:
Sorry, I have forgot to answer to it. 

what do you think about it: 

instead of '-al' for alias, use:  '-s'  or '-a'  
instead of '-id' for ignore description, use:   '-x' 

What do you prefer?